include/configs/ts7180.h

```diff
@@ -57,19 +55,7 @@
 	"fdtfile=imx6ul-ts7180.dtb\0" \
 	"console=ttymxc0,115200\0" \
 	"ethact=ethernet@2188000\0" \
-	"chrg_pct=60\0" \
-	"silochargeon=silabs scaps disable;" \
-		"if test $silopresent = '1';" \
-			"then if test $jpnochrg = 'off';" \
-				"then silabs scaps enable;"\
-			"fi;"\
-		"fi;\0" \
-	"silowaitcharge=if test $silopresent = '1';" \
-		"then if test $jpnochrg = 'on';" \
-			"then echo 'NO CHRG jumper is set, not waiting';" \
-			"else silabs scaps wait pct ${chrg_pct};" \
-		"fi;" \
-	"fi;\0" \
+	"silo_chrg_pct=60\0" \
```

Comment:
I'm not sure what the current paradigm is, but, with these changes is there:
1) A way to enable charging without having to set the `silo_chrg_pct` and wait for it?
2) Not have waiting for charge enabled out of the box so the first boot experience isn't the user waiting ~60 s for the platform to actually turn on.

Reply:
1) Same as the last, but:
2) We do have a no charge jumper on this board.  I'll add a preboot in the next pass that sets the charge percent to 0.